### PR TITLE
Fix urllib3 install procedure

### PR DESCRIPTION
**Pull request by pquentin**
We switched from setuptools to Flit, and running setup.py is deprecated anyway.

### CODE REVIEW

File: projects/urllib3/build.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #
 ################################################################################
 
-python3 setup.py install
+pip install .
```
Comment (radarhere) [inline projects/urllib3/build.sh:18]:
```suggestion
python3 -m pip install .
```
I expect this should fix the two failing tests - https://github.com/google/oss-fuzz/runs/4990624155#step:8:156
I also think it's better practice to invoke pip through Python - https://stackoverflow.com/a/67084664/4093019.

Comment (pquentin) [reply projects/urllib3/build.sh]: Thanks, I should have done that from the start indeed

**Comment by pquentin** [comment]
urllib3 installs correctly, and the failures don't appear to be related:

```
('/tmp/not-out/tmp12nvs6tl/fuzz_data_loader', CompletedProcess(args=['bad_build_check', '/tmp/not-out/tmp12nvs6tl/fuzz_data_loader'], returncode=1, stdout=b'BAD BUILD: /tmp/not-out/tmp12nvs6tl/fuzz_data_loader seems to have either startup crash or exit:\n/tmp/not-out/tmp12nvs6tl/fuzz_data_loader -rss_limit_mb=2560 -timeout=25 -seed=1337 -runs=4 < /dev/null\nTraceback (most recent call last):\n  File "post-processing/fuzz_data_loader.py", line 28, in <module>\nModuleNotFoundError: No module named \'fuzz_cfg_load\'\n[80] Failed to execute script fuzz_data_loader\n', stderr=b''))
BAD BUILD: /tmp/not-out/tmp12nvs6tl/fuzz_data_loader seems to have either startup crash or exit:
/tmp/not-out/tmp12nvs6tl/fuzz_data_loader -rss_limit_mb=2560 -timeout=25 -seed=1337 -runs=4 < /dev/null
Traceback (most recent call last):
  File "post-processing/fuzz_data_loader.py", line 28, in <module>
ModuleNotFoundError: No module named 'fuzz_cfg_load'
[237] Failed to execute script fuzz_data_loader
```

**Comment by DavidKorczynski** [comment]
Hi @pquentin - thanks for this, I believe @jonathanmetzman already solved this but is waiting for the base images to update. @jonathanmetzman this PR lgtm besides the fuzz-introspector issue

**Review by DavidKorczynski — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
CI issue should be fixed with this PR https://github.com/google/oss-fuzz/pull/7199

**Pull request merged**